endpointmanager/pkg/nppesquerier/populatenpicontacts.go

```diff
@@ -100,6 +104,53 @@ func isValidURL(url string) bool {
 	return urlmatched && !emailmatched
 }
 
+func removeNestedDoubleQuotesFromCSV(filename string) (error, string) {
```

Comment:
i see that the line you are testing in the test file originally contains nested double quotes. can you say that somewhere in the test to indicate that you are testing that this function is doing what is expected?

Reply:
https://github.com/onc-healthit/lantern-back-end/pull/56/files#diff-082e02182700664adbe8904cd5c4256fR132 you mean like i mention here or more?